recipes/blaze/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "blaze" %}
+{% set version = "0.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/blaze/blaze.git
+  gti_tag: {{ version }}
```

Comment:
I fixed the typo. The tarball isn't available as they didn't cut a release for that tag and they also don't have a [release on pypi](https://github.com/blaze/blaze/issues/1654). :'(